docs/src/index.md

```diff
@@ -123,6 +123,30 @@ julia> ZonedDateTime(he)
 
 ### Comparisons
 
+#### Equality
+
+Two `AbstractInterval`s are considered equal if they have identical left and right
+endpoints (taking `Inclusivity` into account):
```

Comment:
It would be really great to have docstrings for `LeftEndpoint` and `RightEndpoint`

Reply:
Ya, that guy is a slacker ;). I'll open an issue for this as I don't think I'll have the time at the moment: https://github.com/invenia/Intervals.jl/issues/28